src/configuration-parser.ts

```diff
@@ -19,6 +19,8 @@ export class ConfigurationParser {
             pending: "* ",
             successful: ConfigurationParser.isWindows ? "\u221A " : "✓ ",
         },
+        // tslint:disable-next-line:no-unbound-method
+        print: console.log,
```

Comment:
My point is that `console.log` was [already instrumented](https://github.com/bcaudan/jasmine-spec-reporter/blob/master/spec/helpers/test-helper.ts#L76) for non-custom-print tests, so I wondered why do we need to also mock it.

My understanding is that we need to do that because we are referencing directly the `console.log` function in:
```
print: console.log
```
Then, we need to instrument the `console.log` function before the execution of `configuration-parser.ts`.

If we use a console.log wrapper for the print function:
```
print: stuff => console.log(stuff)
```
Then, we would not have the need to early mock the log function and the previous  instrumentation should be enough.